scripts/ci/test/test-website-e2e.sh

```
@@ -0,0 +1,65 @@
+    echo "Waiting for website..."
+    sleep "$interval"
+  done
+
+  echo "Website is online"
+}
+
+load_env
+bash "$script_dir/wait-for-backend.sh"
+wait_for_website
+
+export WEBSITE_URL="$(get_client_base_url)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)